### ISSUE TITLE
New pull request does not run build task

### ISSUE THREAD

**Issue by domoritz** [marcoroth/dependabot-bump-together-action#18]
The action create a new pull request but it doesn't build: https://github.com/domoritz/csv2parquet/pull/96. Could that be because the GitHub token doesn't allow running builds on prs created from it?

**Comment by marcoroth** [comment]
Hey @domoritz, it might be because you need to approve the run of "first time contributors". 

But looking at the PR you linked, it seems like it worked out?

**Comment by domoritz** [comment]
The build worked because I merged main which then triggered the build. There was no build for [d124518](https://github.com/domoritz/csv2parquet/pull/96/commits/d1245187a3935ed7e3078fdbdfa856cb4366c0a7) unfortunately.

**Comment by marcoroth** [comment]
I just saw you don't have a `on` `push` or `pull_request` trigger setup for the action:
https://github.com/domoritz/csv2parquet/blob/71732cb07712b089844d49f10a3d276e46e86aa5/.github/workflows/dependebot.yml#L3-L6

Currently, as configured, it would just run automatically every day at midnight or if you manually trigger the workflow via the GitHub UI.

![Screenshot 2022-08-15 at 19 17 48](https://user-images.githubusercontent.com/6411752/184683952-aede94f1-be8a-489f-ae6e-6c6d2857d7de.png)

**Comment by marcoroth** [comment]
Oh wait, nevermind. I just understood what you mean now. It's not about running this action but your `build` action.

**Comment by marcoroth** [comment]
But then you need to tweak your `rust.yml` for it to run in the right condition. I guess it might be because of the `branches` config here: https://github.com/domoritz/csv2parquet/blob/278b2f9a8f88f4e9406f708690e29442f3e95ed5/.github/workflows/rust.yml#L5-L8

**Comment by domoritz** [comment]
Doesn't that trigger mean we run the action on prs into main?

**Comment by marcoroth** [comment]
I wonder if it's because just the `pull_request` one would trigger in this case. And because it's not a `push` to `main` it's not being run. Not sure though.

**Comment by domoritz** [comment]
When I pushed to the branch, the build triggered, though. I suspect the issue is that actions cannot trigger other action runs without a PAT.

**Comment by marcoroth** [comment]
Hmm, I'm afraid that's something I can't change on the Action level.

**Comment by domoritz** [comment]
Understood. It would be good to document what the right tokens/setup is, though.

**Comment by marcoroth** [comment]
Yeah, I'm happy to add that or to merge a PR which adds it. Do you know which permissions the token needs?

**Comment by domoritz** [comment]
It seems like any PAT will do: https://github.com/orgs/community/discussions/25702. I would expect everyone to run into this issue, though, not just me.
